### ISSUE TITLE
Problem with dirty flag during initialisation

### ISSUE THREAD

**Issue by sigMacity** [candreoliveira/ngMask#110]
Using the following HTML the $dirty flag is clobbered on initialisation:

**<input type="text" class="form-control" name="q" mask="9999/19/39" mask-clean="true" mask-restrict="accept" data-ng-model="someDate" required="">**

I'm guessing it is related to mask-clean being set to true, though I haven't traced the code through.

**Comment by sigMacity** [comment]
...and here is the code!
`<input type="text" name="thedate" mask="9999/19/39" mask-clean="true" mask-restrict="accept" ng-model="theDate" required></input>`

**Comment by sigMacity** [comment]
The dist folder isn't up-to-date - it appears the bug has been corrected in **src/directives**, but not in **dist**.

**Comment by fsarlenga** [comment]
Yes, I opened an issue about the dist folder not being up to date, issue #109
